### PR TITLE
Update cached HOC metric values.

### PR DESCRIPTION
**Pull request by ashercodeorg**

### CODE REVIEW

File: lib/cdo/properties.rb [hunk 1]
```diff
@@ -36,123 +36,123 @@ def self.delete(key)
...
+      'Italy' => 4_849_916,
+      'Taiwan' => 3_824_526,
+      'Spain' => 3_568_500,
+      'Poland' => 3_386_767,
+      'Brazil' => 3_125_785,
+      'Mexico' => 2_478_498,
+      'India' => 2_317_183,
+      'China' => 2_065_029,
+      'Republic of Korea' => 2_021_896,
       'Korea, Republic of' => 1_883_003,
```
Comment (ashercodeorg) [inline lib/cdo/properties.rb:132]: This seems to be a change of name. Let me investigate...

File: lib/cdo/properties.rb [hunk 2]
```diff
@@ -36,128 +36,128 @@ def self.delete(key)
...
+      'Portugal' => 694_856,
+      'Malaysia' => 664_502,
+      'Egypt' => 657_018,
+      'Belgium' => 644_326
     },
-    # The count was reset to 0 in June 2016 as the result of moving to HOC2016
-    # from HOC2015.
+    # The count was reset to 0 in June 2017 as the result of moving to HOC2017
+    # from HOC2016.
     'total_hoc_count' => 0,
```
Comment (ashercodeorg) [inline lib/cdo/properties.rb:159]: Note that we have 255 events for HOC 2017 in the DB at this time. That said, I'm going to "ignore" them in this update.

**Comment by tanyaparker** [comment]
By the way, do you have a list of your tasks like this somewhere? To hand off to your successor?

**Comment by ashercodeorg** [comment]
@bencodeorg and I are trying to compile a list of stuff. That said, one-off tasks like this are unlikely to make the list, I simply do them when I happen to stumble upon them. Here, for example, I ran across this as part of looking into SOLR deprecation.

**Comment by tanyaparker** [comment]
Maybe not the "most important tasks" list but I think it should still be done and otherwise would be forgotten if not for you.

**Comment by breville** [comment]
Is there documentation on how to do this refresh?

**Comment by ashercodeorg** [comment]
Grab the values from the DB and replace the values here appropriately. E.g., from `pegasus-console`, do
```
require 'cdo/properties'
Properties.get_user_metrics
```
for the numbers in lines 43 to 46.

**Pull request merged**